src/pages/axa-ch-main/commercial-hero-banner.js

```diff
@@ -0,0 +1,20 @@
+import { html } from 'lit-html';
+
+export default html`
+  <axa-commercial-hero-banner
+    variant="dark"
+    imagesource="https://brandcenter.axa.ch/m/4df4627fddd0534a/WIDE_1920_560-AXA_DE_DANNY_web.jpg"
+  >
+    <h2 slot="category">
+      Jetzt Haushalt und Privathaftpflicht versichern und profitieren
```

Comment:
do we want everything in german?

Reply:
No, nothing in german. I forgot to change that here.